core/src/main/java/com/xatkit/core/recognition/IntentRecognitionProviderFactory.java

```diff
@@ -35,6 +36,20 @@
  */
 public class IntentRecognitionProviderFactory {
 
+    /**
+     * The database model that will be used for this instance of Xatkit.
+     * <p>
+     * This property is optional, and it's default value is set to "MAPDB" if not specified
+     */
+    private static final String XATKIT_DATABASE_KEY = "xatkit.database.model";
```

Comment:
Renaming it DATABASE_MODEL_KEY. We don't use the XATKIT prefix in properties constants (see for example the constants in DialogFlowConfiguration).
I'll take care of it

Reply:
And additional note: all these constants should be public: this way it's possible to use them directly from client code.